projects/L4T/packages/tegra-bsp/package.mk

```diff
@@ -1,177 +1,432 @@
-    mkdir multimedia_api
-    cd multimedia_api
-    ar x ../nvidia-l4t-jetson-multimedia-api_32.3.1-20191209225816_arm64.deb
-    mkdir data
-    cd data
-    tar xf ../data.tar.bz2
-    cd ..
-    rm *.tar* debian-binary
-    mv data/* ./
-    rmdir data
-    cp -Pv ${PKG_DIR}/assets/NvV4l2ElementPlane.cpp usr/src/jetson_multimedia_api/samples/common/classes/
-
-    mkdir -p ${PKG_BUILD}/install
-    mkdir -p ${INSTALL}  
-    cd ${PKG_BUILD}/install
-
-    # extract BSP files
-    tar xf ../nv_tegra/config.tbz2
-    tar xf ../nv_tegra/nvidia_drivers.tbz2
-    tar xf ../nv_tegra/nv_tools.tbz2
-    tar xf ../nv_tegra/nv_sample_apps/nvgstapps.tbz2
-
-    # move lib/* to usr/lib to avoid /lib symlink conflicts
-    mv lib/* usr/lib/
-    rm -r lib
-
-    # Same for sbin/
-    mv usr/sbin/* usr/bin/
-
-    # Move usr/lib/aarch64-linux-gnu/* usr/lib/tegra/* and usr/lib/tegra-egl/* to usr/lib/ for Lakka to resolve libs correctly
-    mv usr/lib/aarch64-linux-gnu/* usr/lib/
-    mv usr/lib/tegra-egl/* usr/lib/
-    mv usr/lib/tegra/*  usr/lib/
-    rm -r usr/lib/{tegra-egl,tegra,aarch64-linux-gnu}
-
-    # Remove unneeded files
-    rm -rf usr/lib/ld.so.conf usr/lib/ubiquity \
-  	etc/{systemd,NetworkManager,fstab,lightdm,nv-oem-config.conf.t210,skel,wpa_supplicant.conf,enctune.conf,nv,nvphsd.conf,nvpmodel,xdg} \
-        usr/bin
-    rm -rf etc/systemd etc/sysctl.d etc/hostname etc/hosts etc/modprobe.d etc/modules-load.d var/nvidia
-
-    if [ "${DEVICE}" = "Switch" ]; then
-      #We package custom versions of this in switch-alsa-ucm-configs package
-      rm -rf usr/share/alsa/init/postinit
-    fi
-    
-    # Move udev from etc/ to usr/lib/
-    cp -PRv etc/udev usr/lib/
-    rm -rf etc/udev
-
-    # Refresh symlinks
-    cd usr/lib/
-    ln -sfn libcuda.so.1.1 libcuda.so
-    mv libdrm.so.2 libdrm_nvdc.so
-    ln -sfn libdrm_nvdc.so libdrm.so.2
-    ln -sfn libnvbufsurface.so.1.0.0 libnvbufsurface.so
-    ln -sfn libnvbufsurftransform.so.1.0.0 libnvbufsurftransform.so
-    ln -sfn libnvbuf_utils.so.1.0.0 libnvbuf_utils.so
-    ln -sfn libnvdsbufferpool.so.1.0.0 libnvdsbufferpool.so
-    ln -sfn libnvid_mapper.so.1.0.0 libnvid_mapper.so
-    ln -sfn libnvv4l2.so libv4l2.so.0.0.999999
-    ln -sfn libnvv4lconvert.so libv4lconvert.so.0.0.999999
-    ln -sfn libv4l2.so.0.0.999999 libv4l2.so.0
-    ln -sfn libv4l2.so.0 libv4l2.so
-    ln -sfn libv4lconvert.so.0.0.999999 libv4lconvert.so.0
-    ln -sfn libv4lconvert.so.0 libv4lconvert.so
-    ln -sfn libnvgbm.so libgbm.so.1
-    ln -sfn libnvidia-egl-wayland.so libnvidia-egl-wayland.so.1
-
-    mkdir -p aarch64-linux-gnu/libv4l/plugins/nv
-    cd aarch64-linux-gnu/libv4l/plugins/nv
-    ln -sfn ../../../../libv4l2_nvvideocodec.so libv4l2_nvvideocodec.so
-    ln -sfn ../../../../libv4l2_nvvidconv.so  libv4l2_nvvidconv.so
-    cd ../../../../
+make_host() {
+  if [ -d "${PKG_BUILD}/host_install" ]; then
+    rm -r "${PKG_BUILD}/host_install"
+  fi
+  mkdir -p "${PKG_BUILD}/host_install"
```

Comment:
remove these quotes, I think the buildsystem checks for a space in the path to build root and does not allow building in such path.